uktrade/data-hub-frontend
# Add company business details page based on React

## Description of change

Add new business details page created using React, it will replace the current page in a next PR.

## Test instructions

Navigate to `companies/COMPANY_ID/business-details2`

Check if this page works like the old one.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
